platform/src/main/resources/example.anchor-config.yaml

```diff
@@ -99,7 +102,7 @@ metrics:
   run_interval: 30
 
 events:
-  enabled: true
+  enabled: false
```

Comment:
Does this affect any other testing or deployments?

Reply:
No, this config not used anywhere